src/special_matrices.jl

```diff
@@ -343,10 +343,9 @@ function gram_from_n_r_vectors(M)
 
 end
 
-function gram_matrix_toy_model(n::Int, x::Real)
+function gram_matrix_one_param(n::Int, x::Float64)
```

Comment:
why this restriction? you could 
`@argcheck x>=0 && x<= 1`

Reply:
I mean why did you change from Real to Float64? Generally I think that types should not be specified unless the code requires it to make sense, I think it is not seen as very Julia-nic